Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerPerformanceTrackerTests.swift

```diff
@@ -643,12 +693,13 @@ class SentryUIViewControllerPerformanceTrackerTests: XCTestCase {
 
         let children: [Span]? = Dynamic(tracer).children as [Span]?
 
-        //First Controller viewDidLoad
-        //Second Controller root span
-        //Second Controller viewDidLoad
-        //Third Controller root span
-        //Third Controller viewDidLoad
-        XCTAssertEqual(children?.count, 5)
+        // First Controller initial_display
```

Comment:
Huh, why weren't we expecting any TTID before here? Should we also be expecting a TTFD, or is that already tested elsewhere?

Reply:
> why weren't we expecting any TTID before here

That's the bug 😄. I have no clue. If we would have, we wouldn't have this bug.

> Should we also be expecting a TTFD, or is that already tested elsewhere?

No, because `enableWaitForFullDisplay` isn't enabled for this test. It added another test for TTFD: `test_OnlyViewDidLoadTTFDEnabled_CreatesTTIDAndTTFDSpans`.